models/inorganic_volume_filter/looksrare/inorganic_volume_filter_looksrare.sql

```diff
@@ -272,13 +269,13 @@ trades_enrich as (
 filtered_trades as (
         SELECT 
             t.*, 
-            CASE WHEN mt.filter IS NOT NULL THEN true ELSE FALSE END as mt_filter,
-            CASE WHEN sb.filter IS NOT NULL THEN true ELSE FALSE END as sb_filter,
-            CASE WHEN lv.filter IS NOT NULL THEN true ELSE FALSE END as lv_filter,
-            CASE WHEN hp.filter IS NOT NULL THEN true ELSE FALSE END as hp_filter,
-            CASE WHEN wf.filter IS NOT NULL THEN true ELSE FALSE END as wf_filter,
-            CASE WHEN cb.filter IS NOT NULL THEN true ELSE FALSE END as cb_filter,
-            CASE WHEN cs.filter IS NOT NULL THEN true ELSE FALSE END as cs_filter,
+            CASE WHEN mt.filter IS NOT NULL THEN 'true' ELSE 'false' END as mt_filter,
```

Comment:
We should avoid representing these as strings when they are boolean values.

The code before was correct. My comment meant to say that the `case` was redundant and the code could be more succinct, but it was correctly implemented :) 

(sorry for the confusion!)

I'd make these either:

- `CASE WHEN mt.filter IS NOT NULL THEN true ELSE FALSE END as mt_filter,`
- or `mt.filter IS NOT NULL as mt_filter`

Some people choose the first option to be more explicit, others prefer the one below. Not a big deal IMO, but whichever we do, data type should be bool.

Reply:
okaiee serrr, tyvm will update!